charts/openfaas-cloud/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for openfaas-cloud.
+# This is a YAML-formatted file.
+
+## ofBuilder is the section for of-builder configuration specific settings
+ofBuilder:
+
+  ## names is the name of the deployment for the of-builder component
+  name: of-builder
```

Comment:
I've not seen this done before, or perhaps we keep the name then prefix with the release name?

https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/templates/queueworker-dep.yaml#L11

I wonder if it'd be simpler if the name wasn't templated?

Reply:
Yeah i totally misread the "best practices" doc, change incoming 🚀